src/components/Adyen/Adyen.js

```diff
@@ -288,17 +289,21 @@ const Adyen = ({
     const isBancontactCard =
       selectedPaymentMethodRef?.current?.methodName === 'bancontact_card';
 
-    let checkbox = document.querySelector(`.checkbox-${methodName}`);
+    let checkbox = document.querySelector(`#checkbox-${methodName}`);
+    let checkboxWrapper = document.querySelector(`.checkbox-${methodName}`);
```

Comment:
```suggestion
    let checkboxLabel = document.querySelector(`.checkbox-${methodName}`);
```

Reply:
At this point this element is div (its `CheckboxLegacy`) but I will change it to label